app/src/pages/inside/settingsPage/demoDataGenerator/demoDataGeneratorForm.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import { propTypes, reduxForm } from 'redux-form';
+import { Input } from 'components/inputs/input';
+import classNames from 'classnames/bind';
+import { FormattedMessage } from 'react-intl';
+
+import { BigButton } from 'components/buttons/bigButton';
+import { FieldProvider } from 'components/fields/fieldProvider';
+import { FieldErrorHint } from 'components/fields/fieldErrorHint';
+import { SpinningPreloader } from 'components/preloaders/spinningPreloader';
+
+import styles from './demoDataGenerator.scss';
+
+const cx = classNames.bind(styles);
+
+@reduxForm({
+  form: 'postfix',
```

Comment:
Incorrect form name. 'postfix' it's a field name. Form should named something like 'demoDataGeneratorForm'